README.md

```diff
@@ -12,9 +12,14 @@ Appears to require maven 3.2.5 to build locally.
 Usage
 ------
 
-`mvn org.nixos.mvn2nix:mvn2nix-maven-plugin:mvn2nix`
+```bash
+nix run -f https://github.com/NixOS/mvn2nix-maven-plugin/archive/master.tar.gz \
+    --command mvn2nix > project-info.json
+```
+
+> The _json_ dependency information will by default write to STDOUT.
 
 System properties
 ------------------
 
-* `mvn2nixOutputFile`: Change the name of the file generated by mvn2nix
+* `outputFile`: Set the name of the file generated by mvn2nix, otherwise it is output to STDOUT
```

Comment:
This system property is useful if you want to embed this plugin within a _pom.xml_ of a project.